packages/general-settings/reducer.js

```diff
@@ -0,0 +1,32 @@
+
+import { actionTypes as profileActionTypes } from '@bufferapp/publish-profile-sidebar';
+import keyWrapper from '@bufferapp/keywrapper';
+
+export const actionTypes = keyWrapper('GENERAL', {
```

Comment:
It will improve our data tracking to use a clearer name here:

```suggestion
export const actionTypes = keyWrapper('GENERAL_SETTINGS', {
```